roofit/roofitcore/inc/RooFitDriver.h

```diff
@@ -0,0 +1,143 @@
+#ifndef ROO_FIT_DRIVER_H
```

Comment:
vs ` #define RooFit_Detail_Buffers_h` - could you agree on one format? ;-)

Reply:
Ok, I'l taking the tatter format as in buffers now!